packages/core/src/hydration/annotate.ts

```diff
@@ -159,6 +161,9 @@ function annotateLContainerForHydration(lContainer: LContainer, context: Hydrati
  * @param doc A reference to the current Document instance.
  */
 export function annotateForHydration(appRef: ApplicationRef, doc: Document) {
+  const injector = appRef.injector;
+  const enableI18nHydration = injector.get(IS_I18N_HYDRATION_ENABLED, false);
```

Comment:
nit: I'd propose to name this const similar to the config token:
```suggestion
  const isI18nHydrationEnabled = injector.get(IS_I18N_HYDRATION_ENABLED, false);
```